lib/core/terraform_config/generator.rb

```diff
@@ -14,14 +26,8 @@ def initialize(config:, template:)
       validate_template_kind!
     end
 
-    def filename
-      return "gvc.tf" if kind == "gvc"
-
-      "#{kind.pluralize}.tf"
-    end
-
-    def tf_config
-      config_class.new(**config_params)
+    def tf_configs
```

Comment:
`Generator` can create multiple terraform configuration files, so now we use `#tf_configs` method which returns `{ tf_config_filename => tf_conig_instance }`